packages/react-server/core/ClientController.js

```diff
@@ -764,6 +764,9 @@ class ClientController extends EventEmitter {
 	_createReactServerRootNode(mountNode, index) {
 		var root = document.createElement("div");
 		root.setAttribute(REACT_SERVER_DATA_ATTRIBUTE, index);
+		if (!root.getAttribute("key")) {
+			root.setAttribute("key", index);
+		}
```

Comment:
I think the guard here isn't necessary?  We've just created `root`, so it shouldn't already have a "key" attribute.